corehq/pillows/mappings/xform_mapping.py

```diff
@@ -47,6 +47,10 @@
         'initial_processing_complete': {"type": "boolean"},
         'partial_submission': {"type": "boolean"},
         "#export_tag": {"type": "string", "index": "not_analyzed"},
+        'external_blobs': {
+            'dynamic': False,
+            'type': 'object'
+        },
```

Comment:
@millerdev you have to change this line: https://github.com/dimagi/commcare-hq/blob/3f49c8a5b711d5434dae72f1e26628797e5a76a4/corehq/pillows/mappings/xform_mapping.py#L6

there's another line in the pillowtop tests that needs to be changed as well (should be able to search for the string)